### DISCUSSION TITLE
Does the Dkron can handle misfired jobs when it comes up ?

### DISCUSSION THREAD

**Discussion by bhushanibm** [dkron-io/dkron#1158] [General]
Can the DKRON server/agent work in kubernetes env ? If so, there might be certain times when the DKRON server/agent is down due to some maintenance and if jobs are scheduled at this point of time are going to be missed. When DKRON server/agent service comes back to running, will all the jobs which were missed original since the service were down, will it be re-triggerred ?

**Comment by vcastellm** [comment]
No, Dkron currently doesn't re-trigger any missed jobs during downtime, the goal of the project is to be up as much time as possible. If that doesn't fit your kubernetes setup it is better to go with a traditional server setup.
